qiskit/circuit/library/standard_gates/i.py

```diff
@@ -48,6 +48,11 @@ def __init__(self, label: Optional[str] = None):
         """Create new Identity gate."""
         super().__init__("id", 1, [], label=label)
 
+    def _define(self) -> None:
```

Comment:
Can you update the docstring above to remove the section about `IGate` being defined as a delay and note that it is now explicitly a nop?

Reply:
Oh yeah, thanks for the reminder: done in 95ec70c5d.